docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -10,18 +10,29 @@ The following figure illustrates flex container terminology and alignment axes,
 
 ![Flex container terminology](media/flex-container.jpg)
 
+> [!NOTE]
+> Use FlexLayout instead of FlexContainer class. FlexContainer class is deprecated since Tizen 6.0 and will be supported for the subsequent two releases only.
+
 <a name="layoutexample"></a>
-## Creating a Flexbox Layout
+## Creating a Flexbox layout
 
 The following example shows how to create a gallery-like flexbox layout using the [Tizen.NUI.BaseComponents.FlexContainer](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.FlexContainer.html) object.
 
 **Figure: Flex container example**
 
 ![Flex container example](media/flexbox-demo.jpg)
 
-1.  Create a flex container as the whole view and set its resize policy to fill its parent (the window):
+1.  Add the following namespaces:
 
+    ```csharp
+    using Tizen.NUI;
+    using Tizen.NUI.BaseComponents;
+    using Tizen.NUI.UIComponents;
     ```
+
+2.  Create a flex container as the whole view by setting the resize policy to fill its parent, the window:
```

Comment:
@pkalota  here I have a confusion, can you please help me understand:
- its parent ? 
what is it referring to?
can we write the subject name instead of **its**
Is **its** referring to **the window**?
Could you please rephrase this?
2.  Create a flex container as the whole view by setting the resize policy to fill its parent, the window:

Reply:
Yes, its parent refers to window. I hope the following change gives more clarity.
`2.  Create a flex container as the whole view by setting the resize policy to fill the window:`